app/models/user.rb

```diff
@@ -9,13 +9,13 @@ class User < ActiveRecord::Base
   include Taxonomix
   include DirtyAssociations
   include UserTime
-  audited :except => [:last_login_on, :password, :password_hash, :password_salt, :password_confirmation]
+  audited :except => [:last_login_on, :password, :password_salt, :password_confirmation]
```

Comment:
Removing password_hash will cause the password itself to be stored in the audit details. Changing the UI to hide this is not sufficient because the hash is still stored and will be available via the API etc.

Either exclude the field from being audited and add a new Audit entry when the password is changed containing whatever details are required, or filter out the value of the password hash in AuditExtensions.

This also seems unrelated to the issue/PR reason about requiring the current password. I'd appreciate it if you'd split the auditing changes away from the password requirement, into another ticket and PR.